python/cugraph/tests/test_bfs.py

```diff
@@ -97,13 +97,10 @@ def test_bfs(managed, pool, graph_file):
     cugraph_vid, cugraph_dist = cugraph_call(cu_M, 0)
 
     # Calculating mismatch
-
-    # assert len(base_dist) == len(cugraph_dist)
     i = 0
     j = 0
     while i < len(cugraph_dist):
-        if base_vid[i] == cugraph_vid[i]:
-            assert base_dist[i] == cugraph_dist[i]
-        else:
-            j = j+1
-        i = i+1
+        if base_vid[j] == cugraph_vid[i]:
```

Comment:
Not sure about the logic here. The previous code was supposed to count the number of mismatches in `j`. What does this code do?

Also, there is room for improving the test here as none actually checks the number of mismatches.

Reply:
Xavier and I had a conversation on the changes in the test. The test asserts on every comparison of distance. If we want to track the number of mismatches, the assertion on 104 could be changed to an err counter

Reply:
Renamed variables and moved assert to the end with a counter in https://github.com/rapidsai/cugraph/pull/764/commits/1528aebe9ea39d6ac0060b1068a76f8c2a36056c